### DISCUSSION TITLE
Spatial Calibration "Cannot determine transform between probe and phantom"

### DISCUSSION THREAD

**Discussion by clllng** [PlusToolkit/PlusLib#1056] [Q&A]
Hello,
When I perform the spatial calibration to get the image to Probe transform, I always have the message appearing on the left saying: "cannot determine transform between probe and phantom (ProbeToPhantom transform is invalid). Make sure all markers are in the field of view". But I checked all the markers are visible, which is the case plus I did a test where I let the probe stationary to be sure all markers are visible.
![2023-02-21 184957](https://user-images.githubusercontent.com/117898542/220339475-e7518201-a4fb-435e-9d24-fd710fa69295.png)

I found that when Plus successfully connected to the tracker there were only the stylus and the phantom model in the 3D window,  the probe model was absent.
![2023-02-21 185147](https://user-images.githubusercontent.com/117898542/220339498-503777b8-6f2a-4a5d-9e8f-5d7fee75dfe0.png)

ERRORs were reported:"Unable to read image orientation from configuration file (Rendering tag, DisplayedImageOrientation attribute). Defauting to MF." "|ERROR|036.266000| Render window unavailable when trying to render 2D image.| in E:\D\PSNPb\PlusApp\fCal\vtkPlusImageVisualizer.cxx(320)" 
I have checked the config file and contrasted it with example files carefully , but I failed to solve the problem.
Here is the PLUS config file :
[PlusDeviceSet_fCal_aimposition_2.0 20230220_20230221_154438.txt](https://github.com/PlusToolkit/PlusLib/files/10793242/PlusDeviceSet_fCal_aimposition_2.0.20230220_20230221_154438.txt)

Thanks a lot !

**Comment by r201** [comment]
Hello! Could you please tell me how you handled the "Cannot determine transform between probe and phantom" error? I'm experiencing exactly the same problem and I haven't been able to resolve it. Thank you!
